app/assets/stylesheets/components/page_specific/_floating_chat.scss

```diff
@@ -72,6 +72,9 @@
   &:focus {
     outline: none;
   }
+  &.display-flex {
```

Comment:
I think we should use a more semantic class name than `display-flex`. (See also below on the `display-flex` class).